contracts/exchange-libs/src/reference_functions.ts

```diff
@@ -1,8 +1,9 @@
 import { ReferenceFunctions } from '@0x/contracts-utils';
-import { LibMathRevertErrors } from '@0x/order-utils';
 import { FillResults, Order } from '@0x/types';
 import { BigNumber } from '@0x/utils';
 
+import { LibMathRevertErrors } from '.';
```

Comment:
This pattern is disallowed. Please reference the actual file where this imported item resides in.

Reply:
Fixed in https://github.com/0xProject/0x-monorepo/pull/2321/commits/2bba0e361fcdb63971a15b938fcb604c17965b45